source/reference/methods.txt

```diff
@@ -1334,6 +1334,13 @@ Sharding Methods
      - Returns a boolean to report if the :term:`balancer` is currently
        enabled.
 
+   * - :method:`sh.getShardedDataDistribution()`
+
```

Comment:
**Suggestion** 

You might want to add a comment here that the function definition is documented in both the docs-mongodb-shell and docs-mongodb-internal repos for future updates.